PostFreeSurfer/PostFreeSurferPipeline.sh

```diff
@@ -54,7 +54,9 @@ defaultSigma=$(echo "sqrt(200)" | bc -l)
 
 #TSC:should --path or --study-folder be the flag displayed by the usage?
 opts_AddMandatory '--study-folder' 'StudyFolder' 'path' "folder containing all subjects" "--path"
-opts_AddMandatory '--subject' 'Subject' 'subject ID' "subject label that matches root dir for a single time point processing. For longitudinal experiment root to be identified correctly, must use longitudinal-timepoint and longitudinal-template options."
+
+opts_AddMandatory '--subject-long' 'Subject' 'subject ID' "subject label, may be different from Session"
+opts_AddMandatory '--session' 'Session' 'session ID' "session (timepoint, visit) label." "placeholder" "--subject" #legacy --subject option
```

Comment:
What is `"placeholder"` for?  This is declared as a mandatory argument (which seems correct), and that doesn't accept a default value argument.

Reply:
fixed